src/App.js

```diff
@@ -1,5 +1,33 @@
-function App() {
-  return <div className="App"></div>;
+import { useState } from 'react';
+import { nanoid } from 'nanoid';
+import CardList from './components/CardList';
+import CardForm from './components/CardForm';
+
+function App({ questionText, answerText, onClick }) {
```

Comment:
ich verstehe nicht wo hier die values von questionText und answerText herkommen. es funktioniert zwar aber ich bin mir nicht sicher ob das best practice ist in der app.js etwas zu defragmentieren. Vllt kannst du dir dafür nochmal ne zweite Meinung holen.

Reply:
> ich verstehe nicht wo hier die values von questionText und answerText herkommen. es funktioniert zwar aber ich bin mir nicht sicher ob das best practice ist in der app.js etwas zu defragmentieren. Vllt kannst du dir dafür nochmal ne zweite Meinung holen.

@tRomanowski, habs anders ausprobiert, funktioniert immernoch 👍